lib/mv_opentelemetry/finch.ex

```diff
@@ -25,6 +25,7 @@ defmodule MvOpentelemetry.Finch do
     finch_attributes = [
       {Trace.http_method(), method},
       {Trace.http_scheme(), scheme},
+      {:"http.host", host},
```

Comment:
are the `http.host` and `net_peer_name` redundant ? Can't we just use the `net_peer_name` ?

Reply:
The standard in OTel got stabilised and we were using older names. The proper field names we should use are `net.peer.name` and `http.status_code`, `http.host` and `http.status` are redundant.

We are keeping it redundant to retain data consistency on honeycomb side. Once we have 90 days of proper standard data we can remove the redundant fields or only enable them based on a feature flag.